prql-compiler/src/semantic/std.prql

```diff
@@ -33,6 +30,7 @@ type array = [null]
 
 # Scalar
 type scalar = int || float || bool || text || date || time || timestamp || null
+type scalar_tuple = {scalar..}
```

Comment:
Do I understand correctly this this is "1 dimensional", i.e. no nesting?

Reply:
This name is not ideal - it should be tuple_of_scalars.

I've added `{type..}` as notation for tuple of many `type`s. It was needed to throw an error when a function was passed instead of evaluating to a column.